polkadot/collator/src/lib.rs

```diff
@@ -45,25 +45,31 @@
 //! to be performed, as the collation logic itself.
 
 extern crate futures;
+extern crate polkadot_primitives;
+extern crate polkadot_runtime;
 extern crate substrate_codec as codec;
 extern crate substrate_primitives as primitives;
-extern crate polkadot_runtime;
-extern crate polkadot_primitives;
 
-use std::collections::{BTreeSet, BTreeMap};
+use std::collections::{BTreeMap, BTreeSet};
 
-use futures::{stream, Stream, Future, IntoFuture};
-use polkadot_primitives::parachain::{self, CandidateSignature, ConsolidatedIngress, Message, Id as ParaId};
+use futures::{stream, Future, IntoFuture, Stream};
+use polkadot_primitives::parachain::{
+	self, CandidateSignature, ConsolidatedIngress, Id as ParaId, Message,
+};
 
 /// Parachain context needed for collation.
 ///
 /// This can be implemented through an externally attached service or a stub.
 pub trait ParachainContext {
 	/// Produce a candidate, given the latest ingress queue information.
-	fn produce_candidate<I: IntoIterator<Item=(ParaId, Message)>>(
+	fn produce_candidate<I: IntoIterator<Item = (ParaId, Message)>>(
```

Comment:
I opted here for the spaces (as I felt that this was more common in our code base), but we could switch this to `type_punctuation_density`